projects/core/src/auth/user-auth/events/user-auth-event.builder.ts

```diff
@@ -33,9 +42,17 @@ export class UserAuthEventBuilder {
    * Register a logout event
    */
   protected registerLogoutEvent(): void {
```

Comment:
Adding a comment here, but it applies to another file:
It would be good if we could add a jsdoc on the Login / Logout events stating that it fires only for the user login / logout, and that it _doesn't_ apply to the ASM agent auth.
We can also state that the events are in fact fired when the user is being emulated by an ASM agent.